apis/zalando.org/v1/types.go

```diff
@@ -168,6 +172,19 @@ type RouteGroupRouteSpec struct {
 	Methods []HTTPMethod `json:"methods,omitempty"`
 }
 
+// +k8s:deepcopy-gen=true
+type RouteGroupTLSSpec struct {
+	// Host specifies the list of hosts included in the TLS secret.
+	// The values in this list must match the name/s used in the tlsSecret.
```

Comment:
I understand this is copied from kubectl explain ingress but "name/s" is strange and its unclear what "tlsSecret" references to. Would it bepossible to rephrase somehow?

Reply:
I mean there is no `tlsSecret` anywhere but there is `secretName`.
I also think `name(s)` reads better than `name/s`.
`Host specifies` -> `Hosts specify`